src/dev/flang/tools/fzjava/FZJava.java

```diff
@@ -185,6 +216,33 @@ void execute()
   {
     if (createDestDir())
       {
+        List<String> emptyList = new List<String>();
+        var feOptions = new FrontEndOptions(0,
+                                            _options._fuzionHome,
+                                            true,
+                                            true,
+                                            _options._loadModules,
+                                            _options._moduleDirs,
+                                            emptyList,
+                                            0,
+                                            true,
+                                            true,
+                                            emptyList,
+                                            false,
```

Comment:
`readStdin` is false, so the precondition should not fail.

Reply:
Different issue here: Would be good to add comments like `/* readStdin */ false` to make this more readable.